### PR TITLE
Added third-party cloud storages support

### PR DESCRIPTION
**Pull request by yumikohey**

### CODE REVIEW

File: test/private/teams.spec.ts
Comment (aeunms) [inline test/private/teams.spec.ts:23]: I think this got replaced during a "replace all."

**Review by aeunms — APPROVED** [review]
This looks good to me! I think we're still waiting on a review from Yuri or Sachin though.

File: src/private/files.ts
Comment (vivek3003) [inline src/private/files.ts:303]: nit: `copyMoveFiles` for an API name might be confusing if we chose to make this public.
Can we have `copyFiles` and `moveFiles` as separate functions? Then no need to ask for `isMove` param.

**Review by vivek3003 — APPROVED** [review]

**Review by ydogandjiev — CHANGES_REQUESTED** [review]

File: src/private/files.ts
Comment (erinha) [reply src/private/files.ts:365]: Then can we at least rename it to be `copyOrMoveFiles`?

File: src/private/files.ts
Comment (ydogandjiev) [inline src/private/files.ts:27]: Did you intentionally remove the `@private` comment? This will cause the interfaces/enums to be added to our public docs here which might not be what we want since these APIs won't work for 3P apps:
https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest

File: src/private/files.ts
Comment (ydogandjiev) [inline src/private/files.ts:167]: Add `@private`

File: src/private/files.ts
Comment (ydogandjiev) [inline src/private/files.ts:171]: Add `@private`

File: src/private/files.ts
Comment (ydogandjiev) [inline src/private/files.ts:374]: NIT: This check can never be false because you are providing a default value for `isMove` in the method params.

**Review by ydogandjiev — APPROVED** [review]
Please make sure all new interfaces and enums that you are exporting are marked as `@private` before checking in your changes.

**Pull request merged**
